Globally standardize "the Microsoft Edge Add-ons website" (#3900)
MicrosoftDocs/edge-developer: microsoft-edge/extensions/update/update-extension.md

For an extension that doesn't use Partner Center and isn't at the Microsoft Edge Add-ons store, see [Set an externally installed extension to automatically update](./auto-update.md).

<!-- ====================================================================== -->
## Update an existing extension at the Microsoft Edge Add-ons store

To update an extension that's at the Microsoft Edge Add-ons store:

1.  Go to your [Partner Center developer dashboard](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd) and select the extension that you want to update.

1.  Update either the extension package or the metadata of the extension.  If you update the extension package, make sure that you increase the version number in the manifest file.

1.  After you make the changes, click **Publish** to update your extension listing and start the certification process.  The certification process for an update to an extension can take up to 7 business days.

1.  After the `Status` column displays `In the store`, the extension update is available at the Microsoft Edge Add-ons store.

> After your extension has been created initially, you can update your extension by [Use the REST API to update an extension](./api/using-addons-api.md).

Author: ```suggestion
After your extension has been published, you can update your extension via the REST API; see [Use the REST API to update an extension](./api/using-addons-api.md).
```
